bin/transform.py

```diff
@@ -40,8 +41,27 @@
 COLUMN_WISE = 0 # i.e. axis=0 // axis="columns"
 POPULATE_VALUE = "NA"
 
-def remove_empty_columns(metadata):
-    metadata.dropna(axis="columns", how="all", inplace=True)
+# Special Entries
+NOT_APPLICABLE = "Not Applicable"
+MISSING = "Missing"
+NOT_COLLECTED = "Not Collected"
+NOT_PROVIDED = "Not Provided"
+RESTRICTED_ACCESS = "Restricted Access"
+BLANK = ""
+
+SPECIAL_ENTRIES = [NOT_APPLICABLE, MISSING, NOT_COLLECTED,
+                    NOT_PROVIDED, RESTRICTED_ACCESS, BLANK]
+SPECIAL_ENTRIES_REGEX = ['(?i)^{}$'.format(x) for x in SPECIAL_ENTRIES] # case insensitive
+
+def remove_any_NA_rows(metadata):
+    # If at least one entry in the row is NA,
+    # then remove the whole row.
+    metadata.dropna(axis=0, how="any", inplace=True)
```

Comment:
This is just a small comment, but I'd recommend using the strings `axis="rows"` or `axis="columns"` instead of the numbers if possible.

Also, why in other places in the code you use the named constant `COLUMN_WISE` which has value of `0`. Does the axis variable differ in pandas depending on the function you call?